stable/backup/templates/cronjob.yaml

```diff
@@ -148,6 +136,9 @@ spec:
               subPath: {{ .Values.admin.tlsClientPEM.key }}
             {{- end }}
           volumes:
+        - name: nuosm
+          mountPath: /usr/local/bin/nuosm
+          subPath: nuosm
```

Comment:
the indent here does not match. Since it doesn't work, is it required at all?

Reply:
At this stage, I can think of no purpose for the backup chart.

I can move it to the incubator tree; or just delete it.

Reply:
In actual fact, the template unit tests in integration/template_resources_test.go do actually verify that the stable/backup/template/cronjob.yaml renders correctly.

But yes, with the rewrite of the integration tests in minikube/minikube_base_database_test.go, the stable/backup chart is no longer tested.

Reply:
I think move the original version referencing nuodocker to incubator...